scripts/go_install.sh

```diff
@@ -37,7 +37,7 @@ if [ -z "${GOBIN}" ]; then
   exit 1
 fi
 
-rm "${GOBIN}/${2}"* || true
+rm -f "${GOBIN}/${2}"* || true
```

Comment:
This looks okay to me for this script/context, though I don't know if using the force flag is the best way to do this overall.

Could we redirect the output instead of using the force flag?

Reply:
I used force because it was consistent with how it's used other places in the project. However I can absolutely change it if you'd like. Would you prefer a redirect instead @killianmuldoon?